src/layer/yolov3detectionoutput.cpp

```diff
@@ -168,7 +168,7 @@ int Yolov3DetectionOutput::forward(const std::vector<Mat>& bottom_blobs, std::ve
     std::vector<BBoxRect> all_bbox_rects;
     std::vector<float> all_bbox_scores;
 
-    for (size_t b = 0; b < bottom_blobs.size(); b++)
+    for (int b = 0; b < bottom_blobs.size(); b++)
```

Comment:
ooop ?

Reply:
done

Reply:
It seems to be unchanged and not resolved

Reply:
Oops, sorry about that. I've missed some commits during interative rebase. Done.